extensions/sql-database-projects/README.md

```diff
@@ -22,6 +31,19 @@ Please report issues and feature requests [here.](https://github.com/microsoft/a
 * Start from an existing database by using the `Create Project from Database` from the command palette or database context menu.
 * Start from an OpenAPI/Swagger spec by using the `Generate SQL Project from OpenAPI/Swagger spec` command (Preview).
 
+## Settings
+
+### General Settings
+- `sqlDatabaseProjects.dotnetSDK Location`: The path to the dotnet SDK. If not set, the extension will attempt to find the dotnet SDK on the system.
+- `sqlDatabaseProjects.microsoftBuildSqlVersion`: Version of Microsoft.Build.Sql to use for SDK-style SQL projects. If not set, the extension will use 0.1.9-preview.
+- `sqlDatabaseProjects.netCoreDoNotAsk`: Override option to not ask to install .NET Core SDK when a supported installation is not found.
```

Comment:
Just wondering: at what point do we stop referring to .NET (5+) as ".NET Core"?  Only once Framework is fully in the grave?

Reply:
according to this, I should already be referring to 5+ as just .net https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-5
will update